src/cpu/ppc64/ppc64_gemm_s8x8s32.cpp

```diff
@@ -150,7 +150,7 @@ dnnl_status_t cblas_gemm_s8x8s32_ppc64(int ATflag, int BTflag,
                 }
             }
             for (int i = 0; i < m; ++i) {
-                comparray[i] = out_round<int32_t>(saturate<int32_t>(
+                comparray[i] = cpu::q10n::out_round<int32_t>(cpu::q10n::saturate<int32_t>(
                         ((double)comparray[i]) * alpha * -128.0));
```

Comment:
Reformating to comply with clang-format
```suggestion
                comparray[i] = cpu::q10n::out_round<int32_t>(
                        cpu::q10n::saturate<int32_t>(
                                ((double)comparray[i]) * alpha * -128.0));
```

Reply:
sorry, my bad. Reformatted the code. Thanks